aspnetcore/blazor/class-libraries.md

```diff
@@ -126,3 +126,28 @@ dotnet nuget publish
 ```
 
 When using the `blazorlib` template, static resources are included in the NuGet package. Library consumers automatically receive scripts and stylesheets, so consumers aren't required to manually install the resources. Note that [static assets aren't supported for server-side apps](#static-assets-not-supported-for-server-side-apps), including when a Blazor library (`blazorlib`) is referenced by a server-side app.
+
+## Create a razor class library with static assets
```

Comment:
cc @javiercn

Reply:
It's an asp.net core feature, not a blazor specific feature